schemas.py

```diff
@@ -0,0 +1,4 @@
+from pydantic import BaseModel, ValidationError
+
+class MultipleJokesRequestParams(BaseModel):
+    count: int
```

Comment:
I can see that you included the max number validation in the `multiple_joke` file. This is part of the validation so I would have `MultipleJokesRequestParams` to be responsible.

Have a look at the pydantic validators [here](https://pydantic-docs.helpmanual.io/usage/validators/) and give them a go.

If you struggle take a look at the below:

```
from pydantic import validator

class MultipleJokesRequestParams(BaseModel):
    count: int
    
     @validator("count")
      def is_within_threshold(cls, value):
           if not  [insert your check here]:
                 raise ValueError("some message')
           return value
```

Reply:
I've tried something similar, to be honest, self-doubt got in the way and most likely just had some typo and it did not work or something :D